addons/test_mail/tests/test_mail_activity.py

```diff
@@ -759,3 +761,253 @@ def test_week_grouping(self):
         self.assertEqual(groups[0][groupby], pg_groups["overdue"])
         self.assertEqual(groups[1][groupby], pg_groups["today"])
         self.assertEqual(groups[2][groupby], pg_groups["planned"])
+
+
+class TestActivitySchedule(TestActivityScheduleCommon):
+    """ Test plan and activity schedule
+
+     We test:
+     - activity scheduling on a single record and in batch
+     - plan scheduling on a single record and in batch
+     - plan creation and consistency
+     """
+
+    def check_action_feedback_schedule_plan(self, action, records):
+        if len(records) == 1:
+            self.assertEqual(action, {
+                'type': 'ir.actions.act_window',
+                'res_model': 'res.partner',
+                'res_id': records.id,
+                'name': records.display_name,
+                'view_mode': 'form',
+                'views': [(False, 'form')]
+            })
+        else:
+            self.assertEqual(action, {
+                'type': 'ir.actions.act_window',
+                'res_model': 'res.partner',
+                'name': 'Launch Plans',
+                'view_mode': 'tree,form',
+                'target': 'current',
+                'domain': [('id', 'in', records.ids)],
+            })
+
+    def get_default_deadline(self, activity_type, context=None):
+        return self.env['mail.activity'].with_context(**(context or {}))._calculate_date_deadline(activity_type)
+
+    def check_plan_activities_created(self, plan, on_record,
```

Comment:
If it is an advanced assert, use the "assert" prefix so that we now it is asserting content, like
```suggestion
    def assertActivitiesFromPlan(self, ...)
```

Reply:
Also would gladly have a small docstring :D (hint: use expected_foo if something is an expected output, to distinguish what is the tested content and what is what we expect).